Source/WebKit/UIProcess/_WKTouchEventGenerator.mm

```diff
@@ -249,38 +248,39 @@ - (BOOL)_sendMarkerHIDEventInWindow:(UIWindow *)window completionBlock:(void (^)
     return YES;
 }
 
-- (void)_updateTouchPoints:(CGPoint*)points count:(NSUInteger)count window:(UIWindow *)window
+- (void)_updateTouchPoints:(std::span<CGPoint>)points window:(UIWindow *)window
 {
     HandEventType handEventType;
     
     // The hand event type is based on previous state.
     if (!_activePointCount)
         handEventType = HandEventTouched;
-    else if (!count)
+    else if (points.empty())
         handEventType = HandEventLifted;
-    else if (count == _activePointCount)
+    else if (points.size() == _activePointCount)
         handEventType = HandEventMoved;
     else
         handEventType = HandEventChordChanged;
     
     // Update previous count for next event.
-    _activePointCount = count;
+    _activePointCount = points.size();
 
     // Update point locations.
-    for (NSUInteger i = 0; i < count; ++i)
+    for (size_t i = 0; i < points.size(); ++i)
         _activePoints[i].point = points[i];
     
     RetainPtr<IOHIDEventRef> eventRef = adoptCF([self _createIOHIDEventType:handEventType]);
     [self _sendHIDEvent:eventRef.get() window:window];
 }
 
-- (void)touchDownAtPoints:(CGPoint*)locations touchCount:(NSUInteger)touchCount window:(UIWindow *)window
+- (void)touchDownAtPoints:(CGPoint*)rawLocations touchCount:(NSUInteger)touchCount window:(UIWindow *)window
 {
     touchCount = std::min(touchCount, HIDMaxTouchCount);
+    auto locations = unsafeMakeSpan(rawLocations, touchCount);
```

Comment:
I kind of feel like we want to tighten this up when we can, with a top level method or function that **only** does the conversion to `span`.

When do we use _forge_ vs. _make_ as the word in the function name for creating a `span` from a raw pointer?

Reply:
unsafeForgeSpan was renamed to unsafeMakeSpan yesterday :)

Reply:
For the record, I’m happy with that rename!